### PR TITLE
Separate out the uninstall directions

### PR DESCRIPTION
**Pull request by sdake**
Also fix the 58->56.  Looking at the istio-init chart, only 56
"CustomResourceDefinition" objects are present.  I am not sure where
this number 57 or 58 originates.

### CODE REVIEW

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit 8864549c19a1297cdbcc787731d83c48ce8ff796

https://app.netlify.com/sites/preliminary-istio/deploys/5c7295cf714ea500083fc542

**Comment by ericvn** [comment]
@sdake There were originally 56 CRDs and then we saw in one PR from last week's testing day, it was changed to 58. I hadn't seen the 58 up until that time using snapshot 6, but when using RC0 I did see 58, so Frank had me change them all to 58. I did more testing with snapshot 6, RC0 and the current code and 56 is what I am seeing now.  I do see there another PR for the 56/58 issue https://github.com/istio/istio.io/pull/3380 and in that one I do note two other files that have 58 and also need updating.

File: content/docs/setup/kubernetes/helm-install/index.md [hunk 1]
```diff
@@ -159,11 +138,11 @@ to manage the lifecycle of Istio.
 
     {{< /text >}}
 
-1. Verify that all `58` Istio CRDs were committed to the Kubernetes api-server using the following command:
+1. Verify that all `56` Istio CRDs were committed to the Kubernetes api-server using the following command:
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/helm-install/index.md]:
```suggestion
1. Verify that all `56` Istio CRDs are committed to the Kubernetes api-server using the following command:
```

File: content/docs/setup/kubernetes/helm-install/index.md [hunk 2]
```diff
@@ -176,14 +155,47 @@ to manage the lifecycle of Istio.
 
 1. Uninstall steps:
 
+
+## Uninstalling Istio
+
+### Option 1: Uninstall with Helm via `helm template`
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/helm-install/index.md]: Two headings one after the other are the first indicator that there is a problem with the content's structure. I suggest we split the document into two one document without tiller and another with.
However, I also suggest we do it after 1.1. 
@sdake Do you agree?

**Comment by sdake** [comment]
@andraxylia is solving this one.

**Pull request closed without merging**